2-mandatory/1-create-functions.js

```diff
@@ -33,9 +38,14 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(array, index) {
+  let result = [];
+  array.map((x) => {
+    if (array.indexOf(x) != index) {
```

Comment:
instead of using an if statement inside a map, can you think of an array method which might be more appropriate here? Remember you can also access the current index within the map function `array.map((x,currentIndex) => {`